portfoliostylesheet.css

```diff
@@ -25,21 +25,22 @@ body {
 h1{}
```

Comment:
maybe delete empty class selectors to neaten up unless you are in progress of adding styling